javav2/example_code/stepfunctions/src/main/java/com/example/stepfunctions/CreateStateMachine.java

```diff
@@ -0,0 +1,104 @@
+//snippet-sourcedescription:[CreateStateMachine.java demonstrates how to creates a state machine for AWS Step Functions.]
+//snippet-keyword:[AWS SDK for Java v2]
+//snippet-keyword:[Code Sample]
+//snippet-service:[AWS Step Functions]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[01/28/2021]
+//snippet-sourceauthor:[scmacdon-AWS]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.stepfunctions;
+
+// snippet-start:[stepfunctions.java2.create_machine.import]
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.sfn.SfnClient;
+import software.amazon.awssdk.services.sfn.model.CreateStateMachineRequest;
+import software.amazon.awssdk.services.sfn.model.StateMachineType;
+import software.amazon.awssdk.services.sfn.model.CreateStateMachineResponse;
+import software.amazon.awssdk.services.sfn.model.SfnException;
+import java.io.FileReader;
+import java.io.IOException;
+// snippet-end:[stepfunctions.java2.create_machine.import]
+
+/**
+ * To run this example, you need a JSON file that represents the Amazon States Language definition for the state machine.
+ *
+ * To see an Amazon States Language definition example that you can use in this code example, see 
+ * "Getting started with AWS Step Functions" at https://docs.aws.amazon.com/step-functions/latest/dg/getting-started.html.
+ */
+
+public class CreateStateMachine {
+
+    public static void main(String[] args) {
+        final String USAGE = "\n" +
+                "Usage:\n" +
+                "    CreateStateMachine <jsonFile> <roleARN> <stateMachineName>\n\n" +
+                "Where:\n" +
+                "    jsonFile - A JSON file that represents the Amazon States Language definition of the state machine.\n\n" +
+                "    roleARN - The Amazon Resource Name (ARN) of the IAM role to use for this state machine.\n" +
```

Comment:
IAM -> AWS Identity and Access Management (IAM)

Reply:
fixed